### ISSUE TITLE
Razor reporting error

### ISSUE THREAD

**Issue by shawniverson** [E-F-A/v4#180]
https://forum.efa-project.org/viewtopic.php?p=16418

**Comment by shawniverson** [comment]
```
type=AVC msg=audit(1595191775.522:749): avc:  denied  { create } for  pid=8984 comm="pyzor" name="servers" scontext=system_u:system_r:httpd_sys_script_t:s0 tcontext=system_u:object_r:httpd_var_lib_t:s0 tclass=file permissive=0
type=AVC msg=audit(1595191775.528:750): avc:  denied  { read } for  pid=8981 comm="spamassassin" name=".razor" dev="dm-0" ino=100742085 scontext=system_u:system_r:httpd_sys_script_t:s0 tcontext=unconfined_u:object_r:httpd_var_lib_t:s0 tclass=lnk_file permissive=0
type=AVC msg=audit(1595191775.529:751): avc:  denied  { read } for  pid=8981 comm="spamassassin" name=".razor" dev="dm-0" ino=100742085 scontext=system_u:system_r:httpd_sys_script_t:s0 tcontext=unconfined_u:object_r:httpd_var_lib_t:s0 tclass=lnk_file permissive=0
```

**Comment by shawniverson** [comment]
```
type=AVC msg=audit(1595192332.063:894): avc:  denied  { write open } for  pid=10264 comm="pyzor" path="/var/lib/php/fpm/.pyzor/servers" dev="dm-0" ino=672506 scontext=system_u:system_r:httpd_sys_script_t:s0 tcontext=system_u:object_r:httpd_var_lib_t:s0 tclass=file permissive=0
```

**Comment by shawniverson** [comment]
```
type=AVC msg=audit(1595192487.517:954): avc:  denied  { read } for  pid=10727 comm="pyzor" name="servers" dev="dm-0" ino=672506 scontext=system_u:system_r:httpd_sys_script_t:s0 tcontext=system_u:object_r:httpd_var_lib_t:s0 tclass=file permissive=0
```

**Comment by shawniverson** [comment]
```
type=AVC msg=audit(1595192487.517:953): avc:  denied  { getattr } for  pid=10727 comm="pyzor" path="/var/lib/php/fpm/.pyzor/servers" dev="dm-0" ino=672506 scontext=system_u:system_r:httpd_sys_script_t:s0 tcontext=system_u:object_r:httpd_var_lib_t:s0 tclass=file permissive=
```

**Comment by shawniverson** [comment]
Fixed in commit 45fafebd63501f76525bf1e24cc97c67c7369cde
